roles/openshift_hosted/tasks/registry.yml

```diff
@@ -89,6 +89,7 @@
       docker-registry: default
     session_affinity: ClientIP
     service_type: ClusterIP
+    clusterip: '{{ openshift_hosted_registry_clusterip | default(none) }}'
```

Comment:
I would suggest defaulting to `omit` so the `oc_service` module does the right thing.
```
clusterip: '{{ openshift_hosted_registry_clusterip | default(omit) }}'
```

Reply:
Made the requested change.